tests/test_mgxs_library_nuclides/test_mgxs_library_nuclides.py

```diff
@@ -25,8 +25,12 @@ def _build_inputs(self):
         # Initialize MGXS Library for a few cross section types
         self.mgxs_lib = openmc.mgxs.Library(self._input_set.geometry)
         self.mgxs_lib.by_nuclide = True
-        self.mgxs_lib.mgxs_types = ['transport', 'nu-fission',
-                                    'nu-scatter matrix', 'chi']
+        self.mgxs_lib.mgxs_types = ['total', 'transport', 'nu-transport',
```

Comment:
Also, looks like none of your commits have passed Travis yet, so there may still be a lingering issue. Make sure your version of Pandas is consistent with that used on the Travis image, that is always a hang up.